### ISSUE TITLE
Introduction-and-Usage.Rmd not executable

### ISSUE THREAD

**Issue by sibojan** [billdenney/pknca#36]
Hi Bill,

I think the best way to learn the package is to run it on a local computer. However, some blocks in the Introduction-and-Usage.Rmd is not executable.   For example,  if I force to run the ## Summarization Options block , I got error message. Is it possible for you to show a real running example and how you get that step?

Thank you,

Sibo

**Comment by billdenney** [comment]
It's a fair note that the introduction and usage example isn't runnable, but it is not intended to be run.  It is documentation of a common workflow.  The theophylline example is runnable (see file https://github.com/billdenney/pknca/blob/master/vignettes/Example-theophylline.Rmd).  I have just clarified in the introduction and usage example that it's not runnable and pointed readers to the theophylline example (the link assumes that the package is installed or that you're looking on CRAN-- it won't work from GitHub).

Your comment also made me note a few functions that are not exported, and I am now exporting them (with a commit that I just made).
